>> python-3.8.2 has been loaded.
>> cuda-10.1_cudnn-7.6.5 has been loaded.
WARNING: pip is being invoked by an old script wrapper. This will fail in a future version of pip.
Please see https://github.com/pypa/pip/issues/5599 for advice on fixing the underlying issue.
To avoid this problem you can invoke Python with '-m pip' instead of running pip directly.
WARNING: pip is being invoked by an old script wrapper. This will fail in a future version of pip.
Please see https://github.com/pypa/pip/issues/5599 for advice on fixing the underlying issue.
To avoid this problem you can invoke Python with '-m pip' instead of running pip directly.
WARNING: pip is being invoked by an old script wrapper. This will fail in a future version of pip.
Please see https://github.com/pypa/pip/issues/5599 for advice on fixing the underlying issue.
To avoid this problem you can invoke Python with '-m pip' instead of running pip directly.
WARNING: pip is being invoked by an old script wrapper. This will fail in a future version of pip.
Please see https://github.com/pypa/pip/issues/5599 for advice on fixing the underlying issue.
To avoid this problem you can invoke Python with '-m pip' instead of running pip directly.
WARNING: pip is being invoked by an old script wrapper. This will fail in a future version of pip.
Please see https://github.com/pypa/pip/issues/5599 for advice on fixing the underlying issue.
To avoid this problem you can invoke Python with '-m pip' instead of running pip directly.
WARNING: pip is being invoked by an old script wrapper. This will fail in a future version of pip.
Please see https://github.com/pypa/pip/issues/5599 for advice on fixing the underlying issue.
To avoid this problem you can invoke Python with '-m pip' instead of running pip directly.
WARNING: pip is being invoked by an old script wrapper. This will fail in a future version of pip.
Please see https://github.com/pypa/pip/issues/5599 for advice on fixing the underlying issue.
To avoid this problem you can invoke Python with '-m pip' instead of running pip directly.
WARNING: pip is being invoked by an old script wrapper. This will fail in a future version of pip.
Please see https://github.com/pypa/pip/issues/5599 for advice on fixing the underlying issue.
To avoid this problem you can invoke Python with '-m pip' instead of running pip directly.
WARNING: Skipping page https://pypi.org/simple/attrs/ because the GET request got Content-Type: .The only supported Content-Type is text/html
ERROR: Could not find a version that satisfies the requirement attrs==18.2.0 (from versions: none)
ERROR: No matching distribution found for attrs==18.2.0
srun: error: saliksik-cpu-21: task 5: Exited with exit code 1
    ERROR: Command errored out with exit status 1:
     command: /opt/hpcc/CPU/python/3.8.2/bin/python3.8 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/var/tmp/pip-install-1ujed07_/greenery_65db27a9705c495cb22d76bb2ef3b1a7/setup.py'"'"'; __file__='"'"'/var/tmp/pip-install-1ujed07_/greenery_65db27a9705c495cb22d76bb2ef3b1a7/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /var/tmp/pip-pip-egg-info-grwmdahy
         cwd: /var/tmp/pip-install-1ujed07_/greenery_65db27a9705c495cb22d76bb2ef3b1a7/
    Complete output (11 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/opt/hpcc/CPU/python/3.8.2/lib/python3.8/site-packages/setuptools/__init__.py", line 20, in <module>
        from setuptools.dist import Distribution, Feature
      File "/opt/hpcc/CPU/python/3.8.2/lib/python3.8/site-packages/setuptools/dist.py", line 35, in <module>
        from setuptools import windows_support
      File "/opt/hpcc/CPU/python/3.8.2/lib/python3.8/site-packages/setuptools/windows_support.py", line 2, in <module>
        import ctypes
      File "/opt/hpcc/CPU/python/3.8.2/lib/python3.8/ctypes/__init__.py", line 7, in <module>
        from _ctypes import Union, Structure, Array
    ModuleNotFoundError: No module named '_ctypes'
    ----------------------------------------
WARNING: Discarding https://files.pythonhosted.org/packages/82/69/2c2a77e7034650954e54144ea2b88ec8742a81ac69fbe84da319e0e9112c/greenery-3.1.zip#sha256=e2d98192187bc27caa218ae59d5e34a779ec538ecd3ec839c469f36686421a7e (from https://pypi.org/simple/greenery/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
ERROR: Could not find a version that satisfies the requirement greenery==3.1 (from versions: 1.0, 1.1, 2.0, 2.1, 3.0, 3.1, 3.2, 3.3.1, 3.3.2)
ERROR: No matching distribution found for greenery==3.1
    ERROR: Command errored out with exit status 1:
     command: /opt/hpcc/CPU/python/3.8.2/bin/python3.8 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/var/tmp/pip-install-8vhx08bc/greenery_aae1d320d4cb421aac320b5cee5f2cb8/setup.py'"'"'; __file__='"'"'/var/tmp/pip-install-8vhx08bc/greenery_aae1d320d4cb421aac320b5cee5f2cb8/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /var/tmp/pip-pip-egg-info-6ksx9kat
         cwd: /var/tmp/pip-install-8vhx08bc/greenery_aae1d320d4cb421aac320b5cee5f2cb8/
    Complete output (11 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/opt/hpcc/CPU/python/3.8.2/lib/python3.8/site-packages/setuptools/__init__.py", line 20, in <module>
        from setuptools.dist import Distribution, Feature
      File "/opt/hpcc/CPU/python/3.8.2/lib/python3.8/site-packages/setuptools/dist.py", line 35, in <module>
        from setuptools import windows_support
      File "/opt/hpcc/CPU/python/3.8.2/lib/python3.8/site-packages/setuptools/windows_support.py", line 2, in <module>
        import ctypes
      File "/opt/hpcc/CPU/python/3.8.2/lib/python3.8/ctypes/__init__.py", line 7, in <module>
        from _ctypes import Union, Structure, Array
    ModuleNotFoundError: No module named '_ctypes'
    ----------------------------------------
    ERROR: Command errored out with exit status 1:
     command: /opt/hpcc/CPU/python/3.8.2/bin/python3.8 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/var/tmp/pip-install-a6ljxjph/greenery_311af5fbe16342868e8edafa43712ff1/setup.py'"'"'; __file__='"'"'/var/tmp/pip-install-a6ljxjph/greenery_311af5fbe16342868e8edafa43712ff1/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /var/tmp/pip-pip-egg-info-k14ocqgf
         cwd: /var/tmp/pip-install-a6ljxjph/greenery_311af5fbe16342868e8edafa43712ff1/
    Complete output (11 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/opt/hpcc/CPU/python/3.8.2/lib/python3.8/site-packages/setuptools/__init__.py", line 20, in <module>
        from setuptools.dist import Distribution, Feature
      File "/opt/hpcc/CPU/python/3.8.2/lib/python3.8/site-packages/setuptools/dist.py", line 35, in <module>
        from setuptools import windows_support
      File "/opt/hpcc/CPU/python/3.8.2/lib/python3.8/site-packages/setuptools/windows_support.py", line 2, in <module>
        import ctypes
      File "/opt/hpcc/CPU/python/3.8.2/lib/python3.8/ctypes/__init__.py", line 7, in <module>
        from _ctypes import Union, Structure, Array
    ModuleNotFoundError: No module named '_ctypes'
    ----------------------------------------
    ERROR: Command errored out with exit status 1:
     command: /opt/hpcc/CPU/python/3.8.2/bin/python3.8 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/var/tmp/pip-install-1g7t1a4u/greenery_17224199216846e4b9602ac45a4c1b00/setup.py'"'"'; __file__='"'"'/var/tmp/pip-install-1g7t1a4u/greenery_17224199216846e4b9602ac45a4c1b00/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /var/tmp/pip-pip-egg-info-osdiz6t_
         cwd: /var/tmp/pip-install-1g7t1a4u/greenery_17224199216846e4b9602ac45a4c1b00/
    Complete output (11 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/opt/hpcc/CPU/python/3.8.2/lib/python3.8/site-packages/setuptools/__init__.py", line 20, in <module>
        from setuptools.dist import Distribution, Feature
      File "/opt/hpcc/CPU/python/3.8.2/lib/python3.8/site-packages/setuptools/dist.py", line 35, in <module>
        from setuptools import windows_support
      File "/opt/hpcc/CPU/python/3.8.2/lib/python3.8/site-packages/setuptools/windows_support.py", line 2, in <module>
        import ctypes
      File "/opt/hpcc/CPU/python/3.8.2/lib/python3.8/ctypes/__init__.py", line 7, in <module>
        from _ctypes import Union, Structure, Array
    ModuleNotFoundError: No module named '_ctypes'
    ----------------------------------------
    ERROR: Command errored out with exit status 1:
     command: /opt/hpcc/CPU/python/3.8.2/bin/python3.8 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/var/tmp/pip-install-gli3ii4n/greenery_b2566e9b181a4c4b98a6a9f11b35e3c9/setup.py'"'"'; __file__='"'"'/var/tmp/pip-install-gli3ii4n/greenery_b2566e9b181a4c4b98a6a9f11b35e3c9/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /var/tmp/pip-pip-egg-info-cv8hesgv
         cwd: /var/tmp/pip-install-gli3ii4n/greenery_b2566e9b181a4c4b98a6a9f11b35e3c9/
    Complete output (11 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/opt/hpcc/CPU/python/3.8.2/lib/python3.8/site-packages/setuptools/__init__.py", line 20, in <module>
        from setuptools.dist import Distribution, Feature
      File "/opt/hpcc/CPU/python/3.8.2/lib/python3.8/site-packages/setuptools/dist.py", line 35, in <module>
        from setuptools import windows_support
      File "/opt/hpcc/CPU/python/3.8.2/lib/python3.8/site-packages/setuptools/windows_support.py", line 2, in <module>
        import ctypes
      File "/opt/hpcc/CPU/python/3.8.2/lib/python3.8/ctypes/__init__.py", line 7, in <module>
        from _ctypes import Union, Structure, Array
    ModuleNotFoundError: No module named '_ctypes'
    ----------------------------------------
    ERROR: Command errored out with exit status 1:
     command: /opt/hpcc/CPU/python/3.8.2/bin/python3.8 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/var/tmp/pip-install-59vhuiuc/greenery_dbfe2706621b460a9319a6f4954349ce/setup.py'"'"'; __file__='"'"'/var/tmp/pip-install-59vhuiuc/greenery_dbfe2706621b460a9319a6f4954349ce/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /var/tmp/pip-pip-egg-info-hib7kprg
         cwd: /var/tmp/pip-install-59vhuiuc/greenery_dbfe2706621b460a9319a6f4954349ce/
    Complete output (11 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/opt/hpcc/CPU/python/3.8.2/lib/python3.8/site-packages/setuptools/__init__.py", line 20, in <module>
        from setuptools.dist import Distribution, Feature
      File "/opt/hpcc/CPU/python/3.8.2/lib/python3.8/site-packages/setuptools/dist.py", line 35, in <module>
        from setuptools import windows_support
      File "/opt/hpcc/CPU/python/3.8.2/lib/python3.8/site-packages/setuptools/windows_support.py", line 2, in <module>
        import ctypes
      File "/opt/hpcc/CPU/python/3.8.2/lib/python3.8/ctypes/__init__.py", line 7, in <module>
        from _ctypes import Union, Structure, Array
    ModuleNotFoundError: No module named '_ctypes'
    ----------------------------------------
WARNING: Discarding https://files.pythonhosted.org/packages/82/69/2c2a77e7034650954e54144ea2b88ec8742a81ac69fbe84da319e0e9112c/greenery-3.1.zip#sha256=e2d98192187bc27caa218ae59d5e34a779ec538ecd3ec839c469f36686421a7e (from https://pypi.org/simple/greenery/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
WARNING: Discarding https://files.pythonhosted.org/packages/82/69/2c2a77e7034650954e54144ea2b88ec8742a81ac69fbe84da319e0e9112c/greenery-3.1.zip#sha256=e2d98192187bc27caa218ae59d5e34a779ec538ecd3ec839c469f36686421a7e (from https://pypi.org/simple/greenery/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
WARNING: Discarding https://files.pythonhosted.org/packages/82/69/2c2a77e7034650954e54144ea2b88ec8742a81ac69fbe84da319e0e9112c/greenery-3.1.zip#sha256=e2d98192187bc27caa218ae59d5e34a779ec538ecd3ec839c469f36686421a7e (from https://pypi.org/simple/greenery/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
    ERROR: Command errored out with exit status 1:
     command: /opt/hpcc/CPU/python/3.8.2/bin/python3.8 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/var/tmp/pip-install-fpyo0i_x/greenery_3af34c726ad3483d888710c7fe64d578/setup.py'"'"'; __file__='"'"'/var/tmp/pip-install-fpyo0i_x/greenery_3af34c726ad3483d888710c7fe64d578/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /var/tmp/pip-pip-egg-info-_691tquh
         cwd: /var/tmp/pip-install-fpyo0i_x/greenery_3af34c726ad3483d888710c7fe64d578/
    Complete output (11 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/opt/hpcc/CPU/python/3.8.2/lib/python3.8/site-packages/setuptools/__init__.py", line 20, in <module>
        from setuptools.dist import Distribution, Feature
      File "/opt/hpcc/CPU/python/3.8.2/lib/python3.8/site-packages/setuptools/dist.py", line 35, in <module>
        from setuptools import windows_support
      File "/opt/hpcc/CPU/python/3.8.2/lib/python3.8/site-packages/setuptools/windows_support.py", line 2, in <module>
        import ctypes
      File "/opt/hpcc/CPU/python/3.8.2/lib/python3.8/ctypes/__init__.py", line 7, in <module>
        from _ctypes import Union, Structure, Array
    ModuleNotFoundError: No module named '_ctypes'
    ----------------------------------------
WARNING: Discarding https://files.pythonhosted.org/packages/82/69/2c2a77e7034650954e54144ea2b88ec8742a81ac69fbe84da319e0e9112c/greenery-3.1.zip#sha256=e2d98192187bc27caa218ae59d5e34a779ec538ecd3ec839c469f36686421a7e (from https://pypi.org/simple/greenery/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
WARNING: Discarding https://files.pythonhosted.org/packages/82/69/2c2a77e7034650954e54144ea2b88ec8742a81ac69fbe84da319e0e9112c/greenery-3.1.zip#sha256=e2d98192187bc27caa218ae59d5e34a779ec538ecd3ec839c469f36686421a7e (from https://pypi.org/simple/greenery/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
ERROR: Could not find a version that satisfies the requirement greenery==3.1 (from versions: 1.0, 1.1, 2.0, 2.1, 3.0, 3.1, 3.2, 3.3.1, 3.3.2)
WARNING: Discarding https://files.pythonhosted.org/packages/82/69/2c2a77e7034650954e54144ea2b88ec8742a81ac69fbe84da319e0e9112c/greenery-3.1.zip#sha256=e2d98192187bc27caa218ae59d5e34a779ec538ecd3ec839c469f36686421a7e (from https://pypi.org/simple/greenery/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
ERROR: Could not find a version that satisfies the requirement greenery==3.1 (from versions: 1.0, 1.1, 2.0, 2.1, 3.0, 3.1, 3.2, 3.3.1, 3.3.2)
ERROR: Could not find a version that satisfies the requirement greenery==3.1 (from versions: 1.0, 1.1, 2.0, 2.1, 3.0, 3.1, 3.2, 3.3.1, 3.3.2)
ERROR: No matching distribution found for greenery==3.1
ERROR: No matching distribution found for greenery==3.1
ERROR: No matching distribution found for greenery==3.1
ERROR: Could not find a version that satisfies the requirement greenery==3.1 (from versions: 1.0, 1.1, 2.0, 2.1, 3.0, 3.1, 3.2, 3.3.1, 3.3.2)
ERROR: Could not find a version that satisfies the requirement greenery==3.1 (from versions: 1.0, 1.1, 2.0, 2.1, 3.0, 3.1, 3.2, 3.3.1, 3.3.2)
ERROR: No matching distribution found for greenery==3.1
ERROR: No matching distribution found for greenery==3.1
ERROR: Could not find a version that satisfies the requirement greenery==3.1 (from versions: 1.0, 1.1, 2.0, 2.1, 3.0, 3.1, 3.2, 3.3.1, 3.3.2)
ERROR: No matching distribution found for greenery==3.1
srun: error: saliksik-cpu-21: tasks 0-4,6-7: Exited with exit code 1
Traceback (most recent call last):
  File "src/abstracts/gpu_selection.py", line 1, in <module>
    import pycuda.autoinit
ModuleNotFoundError: No module named 'pycuda'
Traceback (most recent call last):
  File "src/abstracts/gpu_selection.py", line 1, in <module>
    import pycuda.autoinit
ModuleNotFoundError: No module named 'pycuda'
Traceback (most recent call last):
  File "src/abstracts/gpu_selection.py", line 1, in <module>
    import pycuda.autoinit
ModuleNotFoundError: No module named 'pycuda'
Traceback (most recent call last):
  File "src/abstracts/gpu_selection.py", line 1, in <module>
    import pycuda.autoinit
ModuleNotFoundError: No module named 'pycuda'
Traceback (most recent call last):
  File "src/abstracts/gpu_selection.py", line 1, in <module>
    import pycuda.autoinit
ModuleNotFoundError: No module named 'pycuda'
Traceback (most recent call last):
  File "src/abstracts/gpu_selection.py", line 1, in <module>
    import pycuda.autoinit
ModuleNotFoundError: No module named 'pycuda'
Traceback (most recent call last):
  File "src/abstracts/gpu_selection.py", line 1, in <module>
    import pycuda.autoinit
ModuleNotFoundError: No module named 'pycuda'
Traceback (most recent call last):
  File "src/abstracts/gpu_selection.py", line 1, in <module>
    import pycuda.autoinit
ModuleNotFoundError: No module named 'pycuda'
srun: error: saliksik-cpu-21: tasks 0-7: Exited with exit code 1
